luatest/runner.lua

```diff
@@ -148,6 +149,8 @@ function Runner.parse_cmd_line(args)
             result.output_file_name = next_arg()
         elseif arg == '--repeat' or arg == '-r' then
             result.exe_repeat = tonumber(next_arg()) or error('Invalid value for -r option. Integer required.')
+        elseif arg == '--group-repeat' or arg == '-gr' then
```

Comment:
Shouldn't it be `--repeat-group`?

Reply:
`-gr` must equal `-g -r` https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

Let's pick a single letter, maybe `-g` or `-R`